lib/main.coffee

```diff
@@ -60,7 +60,7 @@ module.exports =
       allPackages = values[1]
       if outdatedPackages.length > 0
         PackageUpdatesStatusView = require './package-updates-status-view'
-        packageUpdatesStatusView = new PackageUpdatesStatusView(statusBar, outdatedPackages)
+        new PackageUpdatesStatusView(statusBar, outdatedPackages)
```

Comment:
Space pen views call `@initialize` of the extended class from constructor, which seems to be responsible for the magic of presenting itself. https://github.com/atom/settings-view/blob/c06ae51ac092d2a608d4cdff4677eeb2453e3e3d/lib/package-updates-status-view.coffee#L11-L14